### ISSUE TITLE
Separate projects into categories

### ISSUE THREAD

**Issue by rafaelsabino** [pachno/pachno#44]
How difficult is it to separate projects into categories on the home page?

It would be very useful to have this visibility, because when you have more than 50 projects, it ends up being bureaucratic to locate yourself by the name of the project only.

**Comment by zegenie** [comment]
That's a good suggestion, and wouldn't be very difficult to implement. In the meantime, have you tried the subproject functionality  to group similar projects together?

**Comment by rafaelsabino** [comment]
Yes, subprojects are perfect. I say in terms of different projects. When you have around 100 projects in the system, it is a little difficult to locate them if many of them are active. Categories would help to separate the types of projects.
